src/Components/SceneList/SceneList.tsx

```diff
@@ -209,8 +246,18 @@ const SceneList: React.FC<SceneListProps> = ({
                                 event.stopPropagation();
                                 setSelectedScene(item);
                                 setIsSceneDialogOpen(true);
+                                const telemetryEvent =
+                                    TelemetryEvents.Builder.SceneList.UserAction
+                                        .EditScene.Initiate;
                                 sendEventTelemetry({
-                                    name: 'Edit scene - open'
+                                    name: telemetryEvent.eventName,
+                                    customProperties: {
+                                        [telemetryEvent.properties
+                                            .itemIndex]: itemIndex
```

Comment:
itemIndex seems to be a string type in the `TelemetryConstants` file, but it seems it is assigned as a number here. can we change the itemIndex name to itemKey in the telemetry constants?
<img width="514" alt="image" src="https://user-images.githubusercontent.com/45217314/188206900-2220d9f0-7aba-43b8-be1a-ca7bc2d66a8b.png">

Reply:
the const is the property name that is used to log to Application Insights. So yes, the key is a string, but the value can be whatever you want. That's why the custom properties are a `string` to `any` mapping.